contracts/pods/PodManager.sol

```diff
@@ -0,0 +1,136 @@
+// SPDX-License-Identifier: GPL-3.0-or-later
+pragma solidity ^0.8.0;
+
+import {TribeRoles} from "../core/TribeRoles.sol";
+import {OptimisticTimelock} from "../dao/timelock/OptimisticTimelock.sol";
+
+import {IControllerV1} from "../pods/interfaces/IControllerV1.sol";
+import {IMemberToken} from "../pods/interfaces/IMemberToken.sol";
+
+/// @notice Contract used by an Admin pod to manage child pods.
+
+/// @dev This contract is primarily a factory contract which an admin
+/// can use to deploy more optimistic governance pods. It will create an
+/// Orca pod and deploy an optimistic timelock alongside it.
+///
+/// The timelock and Orca pod are then linked up so that the Orca pod is
+/// the only proposer and executor.
+contract PodManager {
```

Comment:
This contract is not named correctly. It should be something like "PodFactory", not PodManager as it does not actually manage the pods after they are created.

Reply:
Yeah that's an artifact of broader plans I originally had for this contract - I was planning for it to be a "gateway" into the pods and managing activities like adding signers, removing signers, granting roles etc. Decided against it in the end + missed renaming. Done now